### PR TITLE
fix UseItemPoint, parameters cleared after release action

### PR DESCRIPTION
**Pull request by Dreamkins**
for https://github.com/gemrb/gemrb/pull/1962

### CODE REVIEW

**Comment by bradallred** [comment]
hmmm, aren't `Action`s refcounted? Ideally this would be fixed by increasing the refcount before calling `ReleaseCurrentAction` and decreasing it at the end, but maybe that should wait until we switch that to something less manual, like `std::shared_ptr` so its automatic.

**Comment by lynxlynxlynx** [comment]
Thanks!

File: gemrb/core/GameScript/Actions.cpp [hunk 1]
```diff
@@ -6864,8 +6864,10 @@ void GameScript::UseItemPoint(Scriptable* Sender, Action* parameters)
 		return;
 	}
 
+	Point targetPos = parameters->pointParameter;
+
 	Sender->ReleaseCurrentAction();
-	act->UseItemPoint(Slot, header, parameters->pointParameter, flags);
+	act->UseItemPoint(Slot, header, targetPos, flags);
```
Comment (Mingun) [inline gemrb/core/GameScript/Actions.cpp:6870]: It is better to document such changes explicitly in the code so them wouldn't be refactored back.

Comment (lynxlynxlynx) [reply gemrb/core/GameScript/Actions.cpp:6870]: It is pretty obvious if you're working on actions code.

Comment (Mingun) [reply gemrb/core/GameScript/Actions.cpp:6870]: And yet, the bug appeared, for all the obvious

Comment (bradallred) [reply gemrb/core/GameScript/Actions.cpp:6870]: I agree with you @Mingun, but I already have a WIP patch to use `shared_ptr` for `Actions` (see #1913) which would solve the possibility of this sort of bug appearing again which is probably better than a comment.

**Pull request merged**